tasktiger_admin/templates/tasktiger_admin/tasktiger_task_detail.html

```diff
@@ -1,39 +1,142 @@
 {% extends 'admin/index.html' %}
-{% block body %}
-
-<h2><a href="../../../">TaskTiger</a> – <a href="../">{{ queue }} ({{ state }})</a></h2>
 
-<pre>{{ task_dumped }}</pre>
-{% if integrations %}
-    <p>Links:</p>
-    <ul>
-    {% for name, url in integrations %}
-        <li><a href="{{ url }}">{{ name }}</a></li>
-    {% endfor %}
-    </ul>
-{% endif %}
-{% if executions_dumped %}
-<p>Executions:</p>
-
-    <ul>
+{% block head %}
+    {{ super() }}
+    <style type="text/css">
+        th {
+            width: 1px;
+            white-space: nowrap
+        }
+    </style>
```

Comment:
Can we also add `summary { cursor: pointer; }`?

Also semicolon is missing after `nowrap` above.